server/chat-commands/info.ts

```diff
@@ -2713,7 +2713,7 @@ export const commands: ChatCommands = {
 				target = this.filter(target)!;
 				if (!target) return this.errorReply(`Invalid code.`);
 			}
-			return `/raw <div class="infobox">${Chat.getReadmoreCodeBlock(target)}</div>`;
+			return `/raw <div class="infobox"><div class="infobox-limited">${Chat.getReadmoreCodeBlock(target)}</div></div>`;
```

Comment:
```suggestion
			return `/raw <div class="infobox-limited">${Chat.getReadmoreCodeBlock(target)}</div>`;
```

doing this just creates two boxes.

Reply:
Also, how do you plan to address it when it's not broadcasting (using sendReplyBox)?

Reply:
It won't create two boxes, I have checked it.